src/timeMachine/selectors/index.ts

```diff
@@ -261,13 +264,53 @@ export const getSymbolColumnsSelection = (state: AppState): string[] => {
   )
 }
 
+/*
+  There are 2 types of custom time:
+  I. Flux duration, including signed and unsigned durations
+     examples: -15m, -24h, +3d, 5mo
+  II. JavaScript time: any valid string argument to the `new Date()` constructor
+     examples: '2022-10-14T20:33:01.433Z', '2022/07/04', '2022-10-06 00:00'
```

Comment:
The JavaScript `new Date()` constructor can take a number or a string. But we're not exposing the JavaScript `new Date()` constructor. We are exposing a date picker that creates timestamps in string format (and not a numbered string).

Reply:
Really? Okay, one more case to handle then... I'll push a change with more tests.

Reply:
Okay, after trying to force it to accept a number, the Flux query drops the windowPeriod for some reason. This seemed strange.

After further research, the reason is because Flux only supports RFC 3339 format for the time ranges. So, we should not take a number such as `1665784301703` as this is not RFC 3339 format.

I will update the comment to reflect that.

Reply:
I got it to work with Unix timestamps, finally.